src/Pos/Ssc/GodTossing/GState.hs

```diff
@@ -86,8 +88,12 @@ instance SscGStateClass SscGodTossing where
 loadGlobalState :: (MonadDB m, WithLogger m) => m GtGlobalState
 loadGlobalState = do
     logDebug "Loading SSC global state"
-    gs <- DB.getGtGlobalState
-    gs <$ logInfo (sformat ("Loaded GodTossing state: " %build) gs)
+    DB.getGtGlobalStateMaybe >>= \case
+        Just gs -> gs <$ logInfo (sformat ("Loaded GodTossing state: " %build) gs)
+        Nothing -> do
+          let vcd = VCD.fromList . toList $ genesisCertificates
+          logInfo $ "GodTossing state not found in database, use genesis certificates"
```

Comment:
"use genesis certificates" is a bit hard to understand. Are you giving the log's readers a command, advising them to do something, or telling them what was done because of the `Nothing`?